### PR TITLE
Install pgcrypto extension in ggircs

### PR DESCRIPTION
**Pull request by wenzowski**
This creates the pgcrypto extension on all postgres masters and workers using the administration credentials.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,39 +35,52 @@ jobs:
...
           command: make provision
           name: Provision the cluster project set
+      - run:
+          command: |
+            rm -rf cas-postgres
+            git clone --single-branch --branch shipit-deploy/wksv3k-dev https://github.com/bcgov/cas-postgres.git
+            pushd cas-postgres || exit 1
+            git submodule update --init
+            make configure GIT_SHA1=`git rev-parse HEAD`
+            make build GIT_SHA1=`git rev-parse HEAD`
```
Comment (matthieu-foucault) [inline .circleci/config.yml]: Why do you need to specify the GIT_SHA1 here?

Comment (wenzowski) [reply .circleci/config.yml]: In `git.mk` we have `GIT_SHA1=$(shell echo "$${CIRCLE_SHA1:-`$(GIT) rev-parse HEAD`}")` which gives precedence to the circle env var over getting the hash from git. We could just change that...

Comment (wenzowski) [reply .circleci/config.yml]: see bcgov/cas-pipeline#18

Comment (matthieu-foucault) [reply .circleci/config.yml]: That PR is now merged, you can remove the GIT_SHA1 from here, and update the submodule to https://github.com/bcgov/cas-pipeline/commit/fced66d2ff08fb7f4c327e0cecf29ae4ef150e89

**Pull request merged**